aip/general/0121.md

```diff
@@ -94,6 +94,33 @@ the standard methods. Custom methods offer the same design freedom as
 traditional RPC APIs, which can be used to implement common programming
 patterns, such as database transactions, import and export, or data analysis.
 
+### Consistency
```

Comment:
"Read-after-write consistency"

Otherwise "consistency" is a very overloaded term.

Reply:
Or maybe "strong consistency".

Do we care about write-after-write consistency also?

Reply:
Definition-wise I was hoping to leverage the term from the CAP theorem: https://en.wikipedia.org/wiki/CAP_theorem: "Every read receives the most recent write or an error." Although I imagine we have to qualify this a bit better.

Or I could title the section "Consistency Guarantees", since that generally implies distributed systems?

write-after-write... I didn't think it really mattered, since one can always validate whether their intended state has been applied and re-read again. 

For those who do want to have guarantee around their resource state's write, I figured AIP-154's etag might be helpful?